st2client/setup.py

```diff
@@ -72,7 +72,7 @@
         "Repository": "https://github.com/StackStorm/st2",
         "Documentation": "https://docs.stackstorm.com",
         "Community": "https://stackstorm.com/community-signup",
-        "Questions": "https://forum.stackstorm.com/",
+        "Questions": "https://github.com/StackStorm/st2/discussions",
```

Comment:
I also noticed that this URL in our old `st2client/setup.py` file was out-of date. Ideally we won't be using this setup.py file, but I went ahead and updated it here as well.